test/nni_test/nnitest/run_tests.py

```diff
@@ -258,6 +279,11 @@ def run(args):
             print('skipped {}, training service {} not match [{}]'.format(
                 name, args.ts, test_case_config['trainingService']))
             continue
+
+        if not match_remoteConfig(test_case_config):
```

Comment:
a little strange if we do not test remote but we need to judge `match_remoteConfig`, maybe put these lines under line 288 is better.